.github/workflows/scripts/publish_client_gem.sh

```diff
@@ -3,37 +3,12 @@
 # WARNING: DO NOT EDIT!
```

Comment:
I don't think these publish scripts are specific to github.

Reply:
My personally opinion is if the script will only run in GHA and is specific to GHA, then it probably belongs in `.github` but otherwise it shouldn't be in `.github`.

Reply:
@daviddavis they are, they use the $GITHUB_WORSKPACE environment variable.  Maybe that can be changed but I'm hesitant to do it without being able to test properly.  Maybe @fao89 can do so once he starts working on testing all of the publish scripts?

Reply:
I think it would be worthwhile to change it. I would be happy to make this change and test it out.